[SHIP-6153] Add all orgnization related endpoint documentation (#42)
softdev15/slate: source/index.html.md

}
```

> ## Organization Endpoint
> You can upload organization data to Shipamax. Organization data can be retrieved by ID, or by a query filter. You can also update an organization's details and delete an organization by ID.
> 
> ### POST
> Using the following endpoint you can create a new organization, you will need to provide a JSON body with all of the required details. Organizations are unique to their `externalIds`.
> 
> https://public.shipamax-api.com/api/v2/Organizations
> 

Reviewer: This doesn't match the style of the other documentation in the same file. No need to include the domain name. The style of the API endpoint immediately before this one, for example, is:
| Endpoint                         | Verb  | Description                                                                       |
| -------------------------------- | ----- | --------------------------------------------------------------------------------- |
| /FileGroups/{id}/validationResult | POST  | Submit a new validationResult making it the lastValidationesult of the FileGroup  | 

The description of the endpoint should also focus on what it _means_, if that makes sense.

```suggestion
## Organization Endpoint
An Organization represents a business or other third-party that is referenced from the documents that Shipamax processes, like the shipper or consignee for a bill of lading. Each Organization must have a unique identifier provided by you, usually the identifier used in your own system, which we call `externalId` and is returned in details of processed documents. However, the `id` in the Organization API calls is the Shipamax specific ID.

| Endpoint                         | Verb  | Description                                                                       |
| -------------------------------- | ----- | --------------------------------------------------------------------------------- |
| /Organizations | GET | Retrieve a list of your Organizations that match a filter  | 
| /Organizations | POST  | Create a new Organization  | 
| /Organizations/{id} | PATCH  | Update an Organization by specifying the fields that have changed  | 
| /Organizations/{id} | DELETE | Remove an Organization from the system  | 
```
There are probably other inconsistencies too.

>Example JSON body to create an organization
```json
{

>     "externalId": "A",

Reviewer: This should match with what is returned in the next example
```suggestion
    "externalId": "FF",
```

    "carrier": false,
    "consignee": true,
    "creditor": true,
    "forwarder": false,
    "debtor": false,
    "shipper": false,
    "active": false,

>     "updated": "2020-01-01"

Reviewer: Shouldn't ever set the `updated` column, we auto fill this in for them.
```suggestion
```

}
```

>The POST endpoint will respond with JSON like this:
```json
{
  "id": 9,
  "externalId": "FF",
  "carrier": false,
  "consignee": true,
  "creditor": true,
  "forwarder": false,
  "debtor": false,
  "shipper": false,
  "active": false,
  "updated": "2020-01-01T00:00:00.000Z",

>   "companyId": 100000

Reviewer: I think we can use the "hidden" property of the LB3 JSON schema to make this not return if you want.

    "shipper": false,
    "active": false
}
```

>The POST endpoint will respond with JSON like this:
```json
{
  "id": 9,
  "externalId": "FF",
  "carrier": false,
  "consignee": true,
  "creditor": true,
  "forwarder": false,
  "debtor": false,
  "shipper": false,
  "active": false,
  "updated": "2020-01-01T00:00:00.000Z"
}
```

**NOTE: The ID returned by the endpoint is needed to be able to create and relate addresses and names to the Organization**
### GET
Retrieve an Organization via `GET` request to `https://public.shipamax-api.com/api/v2/Organizations/${id}`

>The GET endpoint responds with JSON like this:
```json
{
  "id": 2,
  "externalId": "XXXYYY",
  "carrier": true,
  "consignee": true,
  "creditor": true,
  "forwarder": false,
  "debtor": false,
  "shipper": true,
  "active": true,
  "updated": "2021-08-06T09:58:20.384Z"
}
```

### GET (Using Query Filter)

Retrieve Organizations via `GET` request to `https://public.shipamax-api.com/api/v2/Organizations` and sending in a JSON body to return all Organizations matching the given filter body, or none for all Organizations.

> The GET Organizations query filter body requires JSON structured like this:
```json

> {

Reviewer 2: Can we please add an example of a using the GET with filter

Author: This part is the example itself?
